test/suite/stripeSamples.test.ts

```diff
@@ -208,7 +201,9 @@ async function simulateSelectAll() {
   await sleep(500);
   await vscode.commands.executeCommand('workbench.action.acceptSelectedQuickOpenItem');
 
+  // Pause for sample save-as name input box
+
   // Simulate select path
-  await sleep(500);
+  await sleep(1000);
```

Comment:
@gracegoo-stripe 
this is the main reason for the build failure
it has passed 5 times since this fix